sr_example/scripts/sr_example/hand_examples/sr_right_hand_follow_finger_trajectory.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+
+# An example of how to follow a trajectory defined using a set of waypoints for the first finger.
+
+import rospy
+from sr_robot_commander.sr_hand_commander import SrHandCommander
+from sr_utilities.hand_finder import HandFinder
+import geometry_msgs.msg
+import copy
+
+rospy.init_node("hand_example_following_finger_traj", anonymous=True)
+
+hand_finder = HandFinder()
+hand_parameters = hand_finder.get_hand_parameters()
+prefix = hand_parameters.mapping.values()[0]
+hand_serial = hand_parameters.mapping.keys()[0]
+
+# Open fingers
+group_id = "fingers"
+hand_commander = SrHandCommander(name=group_id)
+hand_commander.move_to_named_target("fingers_open")
+
+group_id = "first_finger"
+hand_commander = SrHandCommander(name=group_id)
+
+rospy.sleep(2)
+print("\nMoving Index finger following a set of waypoints")
+
+waypoints = []
+pose = geometry_msgs.msg.Pose()
+
+pose = geometry_msgs.msg.Pose()
+pose.position.x = 0.057038
+pose.position.y = -0.13281
+pose.position.z = 0.37172
+pose.orientation.x = 0.60863
+pose.orientation.y = 0.05909
+pose.orientation.z = -0.044238
+pose.orientation.w = 0.79001
+waypoints.append(copy.deepcopy(pose))
```

Comment:
Does this need to be deepcopy? Surely a new Pose is being created each time... Not sure if this is a problem...